server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/UaaConfigurationTokenKeyTest.java

```diff
@@ -0,0 +1,32 @@
+package org.cloudfoundry.identity.uaa.impl.config;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class UaaConfigurationTokenKeyTest {
+
+  private YamlConfigurationValidator<UaaConfiguration> validator = new YamlConfigurationValidator<UaaConfiguration>(
+      new UaaConfiguration.UaaConfigConstructor());
+
+  private UaaConfiguration createValidator(final String yaml) {
+    validator.setExceptionIfInvalid(true);
+    validator.setYaml(yaml);
+    validator.afterPropertiesSet();
+    return validator.getObject();
+  }
+
+  @Test
+  void testTokenKey() {
+    assertNotNull(createValidator("uaa:\n" +
```

Comment:
I think that's another example of something that would fix better in a text file

Reply:
moved to file

Reply:
@strehle and I agree.